components/properties/src/props.rs

```diff
@@ -44,7 +44,7 @@ impl DataMarker for ErasedNameToEnumMapV1Marker {
 /// use icu::properties::GeneralCategory;
 ///
 /// let lookup =
-///     GeneralCategory::get_name_to_enum_mapper(&icu_testdata::unstable())
+///     GeneralCategory::get_name_to_enum_mapper(&icu_testdata::unstable_no_fallback())
```

Comment:
Question: It's not immediately clear to me why you needed to change from `unstable` to `unstable_no_fallback`

Reply:
Kind of to signal that using this function doesn't require fallback (as the key is a singleton).

I'll actually go and update all examples to use the non-provider methods, just wanted to get your ok on that before doing it.

Reply:
I've reverted this, this module actually still needs work. We need alternative names for
* `get_name_to_enum_mapper`
* `get_enum_to_short_name_mapper`
* `get_enum_to_long_name_mapper`